dbt_project.yml

```
@@ -1,5 +1,5 @@
 name: 'the_tuva_project'
-version: '0.7.17'
+version: '0.7.18'
```

Revert package version back to 0.7.17 since we have not released this version yet.
```suggestion
version: '0.7.17'
```

---

I will correct this in a different PR when I update the dbt docs for this package release.